README.md

```diff
@@ -212,6 +221,64 @@ For renaming instructions in particular, see the XML `remarks` tags on class def
 | 304         | 1         | H323 ID of device                                                                                    | Serial              | ToSIMPL      |
 | 305         | 1         | SIP URI of device                                                                                    | Serial              | ToSIMPL      |
 | 311         | 1         | WebexSendPin                                                                                         | DigitalSerial       | FromSIMPL    |
+| 321         | 1         | WidgetEventData                                                                                         | Serial       | ToSIMPL    |
```

Comment:
Lets edit this to fix formatting in the readme direct in case someone looks without a proper rendering engine

For future reference, Essentials 1.12.2 has a user command that exports the joinmap directly to a markdown file- that's what I used to format the table appropriately.